### PR TITLE
fix(repos): hide disabled repos in settings/repos

### PR DESCRIPTION
**Pull request by cathteng**
We now set a repo to have `ObjectStatus.DISABLED` when a user "deletes" a repository. The current `/settings/repos/` page sends an API request with query param `status=` (empty string) which returns all repos regardless of status. Removing the query param will only return active repos. There is no way to delete a disabled repo on this page, which can be confusing for the user when they've already "deleted" it once before.

### CODE REVIEW

**Review by mifu67 — APPROVED** [review]
LGTM, but you might want to wait for another approval

**Review by EricHasegawa — APPROVED** [review]
fix(repos): hide disabled repos in settings/repos > declaration of independence

**Pull request merged**
